README.md

```diff
@@ -12,9 +12,9 @@ As noted in Paul Irish's "[What Forces Layout](https://gist.github.com/paulirish
 
 ## Compatibility
 
-- Ember.js v3.8 or above
-- Ember CLI v2.13 or above
-- Node.js v10 or above
+* Ember.js v3.24 or above
+* Ember CLI v3.24 or above
+* Node.js v12 or above
```

Comment:
Can you bump this to v14+? (It doesn't make sense to me to do a major release and still support an already EOL'ed Node version)

Reply:
fixed in 9b060a19a1e4698862945f731c2a3f257233ab9a